src/virtualize.js

```diff
@@ -111,7 +111,7 @@ export default function virtualize(MyComponent) {
       }
 
       if (onChangeIndex) {
-        onChangeIndex(index);
+        onChangeIndex(index, indexLatest);
```

Comment:
@oliviertassinari Yes, I can see that. But `virtualize` augments `<SwipeableViews/>`, does it not? I can still define `onChangeIndex` for a `virtualize` HOC and it will still work just not like the documentation says due to this issue.

Maybe this is a documentation issue then, if this is your desired behavior. I was expecting to receive the `indexLatest` as per the documentation in my `virtualize` HOC anyway. I can work around it by looking at `this.state.index` because that would hold the same info, right?